server/controllers/finance/purchases.js

```diff
@@ -302,16 +316,12 @@ function find(options) {
       JOIN user AS u ON u.id = p.user_id
   `;
 
-  filters.dateFrom('dateFrom', 'date');
-  filters.dateTo('dateTo', 'date');
-
   const referenceStatement = `CONCAT_WS('.', '${identifiers.PURCHASE_ORDER.key}', pr.abbr, p.reference) = ?`;
   filters.custom('reference', referenceStatement);
 
-  filters.setOrder('ORDER BY p.date DESC');
```

Comment:
You should probably keep the `setOrder()` around.  We want to make sure only recent records are displayed at the top.